packages/domain/src/entities/FollowPolicy.ts

```diff
@@ -19,4 +40,7 @@ export type OpenFollowPolicy = {
   type: FollowPolicyType.ANYONE;
 };
 
+/**
+ * Use when preparing a follow profile request.
```

Comment:
what you mean by this?

Reply:
I had to leave notes to myself to understand the difference between all follow policy types we have. I can remove that comment or change it if you have a better idea.